bin/asa

```diff
@@ -13,10 +13,12 @@ License:
 
 
 use strict;
+use warnings;
+use List::Util qw/ any /;
```

Comment:
The question is if loading a module saves a significant amount of time over using a slightly inefficient grep. Can you benchmark both ways and see which one is faster?

Reply:
Loading the module caused the 5.8 travis build to fail which I note is a requirement so I reverted back to the grep block form in the second commit.  So ultimately the only thing that changed was I turned on warnings and changed the grep from expression to block form.

Reply:
That's odd because List::Util was core in 5.7.3.

For the warnings, I have the same comment as in #78. I'm not sure about shipping developer settings to end users.

Reply:
'any' is only listed as a suggested addition in 5.8 docs.

Let me know if you make a decision, I'm happy to help in any way I can.